### ISSUE TITLE
csv+jsonfile for paralell + serial mass scanning

### ISSUE THREAD

**Issue by drwetter** [drwetter/testssl.sh#751]
It looks like (can't confirm as it is still running) csvfile output doesn't work correctly:

``EXPERIMENTAL=true ./testssl.sh --file=f5.txt -T --logfile=f5.log --csvfile=f5.csv``

randomly puts out lines like ``cat: /tmp/testssl.70Xmy3/csvfile_00000076.csv: No such file or directory``
In fact ``/tmp/testssl.70Xmy3`` contains only screen logs 
```
files
gost.conf                  term_output_00000033.log   term_output_00000069.log   term_output_00000105.log   term_output_00000141.log
s_client_has.txt           term_output_00000034.log   term_output_00000070.log   term_output_00000106.log   term_output_00000142.log
s_client_starttls_has.txt  term_output_00000035.log   term_output_00000071.log   term_output_00000107.log   term_output_00000143.log
term_output_00000000.log   term_output_00000036.log   term_output_00000072.log   term_output_00000108.log   term_output_00000144.log
term_output_00000001.log   term_output_00000037.log   term_output_00000073.log   term_output_00000109.log   term_output_00000145.log
term_output_00000002.log   term_output_00000038.log   term_output_00000074.log   term_output_00000110.log   term_output_00000146.log
term_output_00000003.log   term_output_00000039.log   term_output_00000075.log   term_output_00000111.log   term_output_00000147.log
term_output_00000004.log   term_output_00000040.log   term_output_00000076.log   term_output_00000112.log   term_output_00000148.log
term_output_00000005.log   term_output_00000041.log   term_output_00000077.log   term_output_00000113.log   term_output_00000149.log
term_output_00000006.log   term_output_00000042.log   term_output_00000078.log   term_output_00000114.log   term_output_00000150.log
term_output_00000007.log   term_output_00000043.log   term_output_00000079.log   term_output_00000115.log   term_output_00000151.log
term_output_00000008.log   term_output_00000044.log   term_output_00000080.log   term_output_00000116.log   term_output_00000152.log
[..]
```

**Comment by drwetter** [comment]
no csvfile f5.csv generated...

**Comment by drwetter** [comment]
for serial scanning it's similar. Difference is that the csvfile is being created but the output contains only the header ``"id","fqdn/ip","port","severity","finding","cve","cwe","hint"``.

For ``--jsonfile`` the content is kind of lacking too (
```
[
,,,,]
```

**Comment by drwetter** [comment]
closed via 041abd57ce8b8c06c9a62f836e853272e69fb294
